syntax ="proto3";

package fixbug;

option cc_generic_services =true;

message ResultCode
{
    int32 errcode=1;//0 正确 非零：error
    bytes errmsg=2;
}

message LoginRequest{
    bytes name=1;
    bytes pwd=2;
}

message LoginResponse{
    ResultCode result=1;
    int32 sucess=2;
}

//注册
message RegisterRequest{
    uint32 id=1;
    bytes name=2;
    bytes pwd=3;
}

message RegisterResponse{
    ResultCode result=1;
    int32 sucess=2;
}
service UserServiceRpc{
    rpc Login(LoginRequest)returns(LoginResponse);
    rpc Register(RegisterRequest)returns(RegisterResponse);
}